typescript/sdk/src/consts/environments/index.ts

```diff
@@ -20,7 +21,7 @@ type HyperlaneCoreAddressMap = LooseChainMap<{
 
 // Export developer-relevant addresses
 export const hyperlaneCoreAddresses = objMap(
-  { ...test, ...testnet2, ...mainnet },
+  { ...testnet2, ...mainnet },
```

Comment:
```suggestion
  { ...testnet2, ...testnet3, ...mainnet },
```
?

Reply:
We actually can't put testnet3 in there until middleware is deployed, since this object is typed to expect e.g. `interchainAccountsRouter`